src/features/rewards/walletPanel/index.tsx

```diff
@@ -0,0 +1,199 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+// Helpers
+import { getLocale } from '../../../helpers'
+
+// Styled Components
+import {
+  StyledWrapper,
+  StyledAttentionScore,
+  StyledAttentionScoreTitle,
+  StyledContainer,
+  StyledScoreWrapper,
+  StyledControlsWrapper,
+  StyledDonateText,
+  StyledIcon,
+  StyledDonateWrapper,
+  StyledToggleWrapper,
+  StyledSelectWrapper
+} from './style'
+
+// Components
+import {
+  Grid,
+  Column,
+  Select,
+  Toggle,
+  ButtonSecondary
+} from '../../../components'
+import { Tokens } from '../'
+import Profile, { Provider } from '../profile/index'
+import ToggleTips from '../toggleTips/index'
+
+// Assets
+const alertIcon = require('./assets/alert')
+const monthlyIcon = require('./assets/monthly')
+
+export interface Props {
+  id?: string
+  platform?: Provider
+  publisherImg?: string
+  publisherName?: string
+  isVerified?: boolean
+  attentionScore?: string | null
+  tipsEnabled: boolean
+  includeInAuto: boolean
+  monthlyAmount: number
+  donationAction: () => void
+  onToggleTips: () => void
+  onAmountChange: () => void
+  onIncludeInAuto: () => void
+}
+
+export default class WalletPanel extends React.PureComponent<Props, {}> {
+  constructor (props: Props) {
+    super(props)
+  }
+
+  publisherInfo () {
+    const publisherTitle = this.props.publisherName || ''
+
+    return (
+      <StyledWrapper>
+        <Profile
+          type={'big'}
+          title={publisherTitle}
+          provider={this.props.platform}
+          src={this.props.publisherImg}
+          verified={this.props.isVerified}
+        />
+      </StyledWrapper>
+    )
+  }
+
+  getSelectTheme () {
+    return {
+      border: 'none',
+      padding: '0px',
+      background: 'inherit'
+    }
+  }
+
+  donationDropDown () {
+    const donationAmounts = [
+      5, 10, 15, 20, 30, 50, 100
+    ]
+    const monthlyAmount = this.props.monthlyAmount || 5
+
+    return (
+      <StyledSelectWrapper>
+        <Select
+          theme={this.getSelectTheme()}
+          value={monthlyAmount.toString()}
+          onChange={this.props.onAmountChange}
+        >
+          {donationAmounts.map((amount: number, index: number) => {
+            return (
+              <div key={`${index}`} data-value={amount.toString()}>
```

Comment:
`key={`${index}`}` -> key={`donationAmount-${index}`} so that is more unique then just a number